### ISSUE TITLE
Papercut: docstring of bluemira.fuel_cycle.tools.legal_limit

### ISSUE THREAD

**Issue by OceanNuclear** [Fusion-Power-Plant-Framework/bluemira#2855]
## Describe the papercut

[`bluemira.fuel_cycle.tools.legal_limit`](https://bluemira.readthedocs.io/en/latest/autoapi/bluemira/fuel_cycle/tools/index.html#bluemira.fuel_cycle.tools.legal_limit):
![image](https://github.com/Fusion-Power-Plant-Framework/bluemira/assets/26902225/1fa045a5-920e-4473-9696-7ac01ebb7386)
I think this needs a little clean-up, and it might be worth breaking up the latex formula into two lines.

Not sure why the denominator has [MeV]eV[J] in it? Does it need clarification?

**Comment by OceanNuclear** [comment]
Also it is missing all of the units (e.g. p_fus is in [MW] instead of the SI [W], which is another issue on its own).

**Comment by OceanNuclear** [comment]
Similar questions exists for bluemira/plasma_physics/reactions.py: not sure why is fusion power expressed as {17.58 [MeV]eV[J]N_{A}[1/mol]} in the denominators?

**Comment by OceanNuclear** [comment]
Need to fix bluemira/plasma_physics/reactions.py `E_DD_fusion` too

**Comment by OceanNuclear** [comment]
When trying to build the docs, `sphinx-build -W documentation/source documentation/build` errors with
<pre><b>reading sources... </b>[  0%] <font color="#A347BA">autoapi/bluemira/_version/index</font>                                                                                                                                                         
<font color="#F66151">Exception occurred:</font>
  File &quot;/home/username/mambaforge/envs/bluemira/lib/python3.8/site-packages/autoapi/directives.py&quot;, line 22, in get_items
    obj = all_objects[name]
KeyError: &apos;bluemira._version.VersioneerConfig&apos;
<font color="#F66151">The full traceback has been saved in /tmp/sphinx-err-wblq703z.log, if you want to report the issue to the developers.</font>
</pre>
Making it difficult to assess whether the modification was valid or not.

I shall look into fixing this error in the next month.

**Comment by je-cook** [comment]
> When trying to build the docs, `sphinx-build -W documentation/source documentation/build` errors with
> 
> reading sources... [  0%] autoapi/bluemira/_version/index                                                                                                                                                         
> Exception occurred:
>   File "/home/username/mambaforge/envs/bluemira/lib/python3.8/site-packages/autoapi/directives.py", line 22, in get_items
>     obj = all_objects[name]
> KeyError: 'bluemira._version.VersioneerConfig'
> The full traceback has been saved in /tmp/sphinx-err-wblq703z.log, if you want to report the issue to the developers.
> Making it difficult to assess whether the modification was valid or not.
> 
> I shall look into fixing this error in the next month.

you need to delete the folder `documentation/source/autoapi/` since you last built the docs we have removed that file and the autoapi cache can't find it.

**Comment by OceanNuclear** [comment]
I came back to this with a fresh set of eyes and can answer my own question:

> not sure why is fusion power expressed as {17.58 [MeV]eV[J]N_{A}[1/mol]} in the denominators?

The answer to that is: these are in the format of `quantity[unit]`.
We have 17.58 as the fusion power in [MeV], and it is multiplied onto a constant to convert from [MeV] into [J]. The N_A is in unit [number/mol] hence the [1/mol].

I think this seems a little redundant because the code has already handled the unit conversion (see the branch ocean/plasma-units-advanced) and we can hide all of that away from the user/docstrings. IMO it's easier to read the code directly than to decipher the docstring in this particular format.

Therefore I have opted to clean these up, removing the conversion factors (e.g. `eV[J]`) and all the units tag (e.g. `[MW]`) since the units are already recorded elsewhere in the same docstrings.
